src/Ifelse.java

```diff
@@ -0,0 +1,199 @@
+import java.util.Scanner;
+
+public class Ifelse {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        // Opdracht 1
+        int number1 = 3;
```

Comment:
Het is overzichtelijker als je je variabelen zo veel mogelijk bovenaan de methode definieert, waar ze netjes bij elkaar staan.